b2sdk/transfer/inbound/download_manager.py

```diff
@@ -48,13 +51,15 @@ def __init__(
         write_buffer_size: int | None = None,
         check_hash: bool = True,
         max_download_streams_per_file: int | None = None,
+        retry_time: int | None = None,
         **kwargs
     ):
         """
         Initialize the DownloadManager using the given services object.
         """
 
         super().__init__(**kwargs)
+        self.retry_time = retry_time or self.DEFAULT_RETRY_TIME
```

Comment:
Also, why need to set instance attribute `self.retry_time`? Is it used anywhere? I thought you just pass `retry_time` to `PARALLEL_DOWNLOADER_CLASS` and `SIMPLE_DOWNLOADER_CLASS` and that's it.